docs(gateway): cleanup references to old services (#1116)
kumahq/kuma-website: app/docs/1.8.x/explore/gateway.md

See the [dedicated section](/docs/{{ page.version }}/deployments/multi-zone) for detailed information.
{% endtip %}

> Now let's define a `MeshGatewayRoute` to forward our traffic based on the
> matched URL path.

Reviewer: ```suggestion
Now, you can define a `MeshGatewayRoute` to forward your traffic based on the
matched URL path.
```
